pwndbg/commands/got.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import subprocess
+import struct
+
+import pwndbg.commands
+import pwndbg.commands.checksec
+import pwndbg.enhance
+import pwndbg.chain
+import pwndbg.which
+
+from pwndbg.color import *
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
```

Comment:
Some nice commands, just for reference/as an examples:
* https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/vmmap.py
* https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/auxv.py
* https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/misc.py